src/libslic3r/PrintConfig.cpp

```diff
@@ -1700,6 +1700,15 @@ void PrintConfigDef::init_fff_params()
     def->tooltip = L("If enable this setting, part cooling fan will never be stoped and will run at least "
                      "at minimum speed to reduce the frequency of starting and stoping");
     def->set_default_value(new ConfigOptionBools { false });
+    
+    def = this->add("dont_slow_down_external_perimeters", coBools);
```

Comment:
To make it shorter and consistent with other parameter, what do you think if we call it
`dont_slow_down_outer_wall`
and `Don't slow down outer walls`?

Reply:
Done :)